.github/ISSUE_TEMPLATE/issue_car.yml

```diff
@@ -0,0 +1,74 @@
+name: Telsa Car Issue
+description: Create a report to help us improve
+title: "Car Issue: "
+labels: ["triage", "car"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Before you open a new issue, search through the existing issues to see if others have had the same problem.
```

Comment:
I've tended to also mention "closed" issues for searching too.

Reply:
@skipishere just wanted to confirm if you wanted to add a statement about searching closed issues too. If not, I can just merge it but wasn't sure what you were planning.

Reply:
Sorry not had a lot of time, will try and do that today 👍

Reply:
Sorry took longer than planed to get back to this, happy to merge this whenever you are